go.mod

```diff
@@ -2,6 +2,8 @@ module terraform-provider-delphix
 
 go 1.17
 
+replace github.com/delphix/dct-sdk-go => /Users/ekaspi/repos/dct-sdk-go
```

Comment:
Remove the ref to local sdk version.

Reply:
Will do. Can you please review https://github.com/delphix/dct-sdk-go/pull/16 so that I can release a new sdk version?